plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -239,6 +250,18 @@ export default class ChatMessage {
     return this.manager?.messages?.objectAt?.(this.index + 1);
   }
 
+  highlight() {
+    this.highlighted = true;
+
+    discourseLater(() => {
+      if (this.isDestroying || this.isDestroyed) {
+        return;
+      }
+
+      this.highlighted = false;
+    }, 2000);
+  }
```

Comment:
I hesitated a long time to have this here, but I think it's ok, after all a "model" on the frontend is not exactly like a model on the backend, that makes it quite easy to highlight any message now.